pos_lot_selection/static/src/js/popup/OrderWidget.js

```diff
@@ -0,0 +1,47 @@
+odoo.define('pos_lot_selection.CustomOrderWidget', function(require) {
+    'use strict';
+
+    const Registries = require('point_of_sale.Registries');
+    const OrderWidget = require('point_of_sale.OrderWidget');
+
+    
+    var CustomOrderWidget = (OrderWidget) => class extends OrderWidget{
+        async _editPackLotLines(event) {
+            const orderline = event.detail.orderline;
+            
+            this.env.session.lots = await this.get_lots(orderline.product, this.env.session);
+            return super._editPackLotLines(event);
```

Comment:
Forgot about this, but maybe we should "reset" the values of lots that are stored after the call to super